scripts/questions_gen.py

```diff
@@ -245,6 +245,9 @@ async def questions_eval(
         res_row["passing"] = result.passing
         res_row["feedback"] = result.feedback
         res_row["score"] = result.score
+
+        # TODO: OLS-619 Evaluation model can return `None` instead of score, so
+        #               it is not possible to convert the value w/o checks
```

Comment:
indentation is off

Reply:
no, I wanted to format the text into paragraph, it's on purpose

Reply:
hm, ok